docs/discover/set-time-filter.asciidoc

```diff
@@ -33,8 +34,12 @@ To set a time filter from the histogram, do one of the following:
 the cursor over the background of the chart--the cursor changes to a plus sign
 when you hover over a valid start point.
 
+To move forwards/backwards in time, click on the arrows to the left or right of the Time Picker:
+
+image::images/time-picker-step.jpg[Move backwards in time]
```

Comment:
Forward and backward don't need the "s". (Though I see it's forwards & backwards in the UI, so maybe best to leave as is. It's not wrong, just less common these days.)  I'd also replace the / with "or". 

"click on" can just be "click".